mix.exs

```diff
@@ -33,7 +33,7 @@ defmodule Demo.Mixfile do
   # Type `mix help deps` for examples and options.
   defp deps do
     [
-      {:phoenix, github: "phoenixframework/phoenix", branch: "v1.4", override: true},
+      {:phoenix, github: "phoenixframework/phoenix", branch: "master", override: true},
```

Comment:
This commit will be removed once this pull request is merged
https://github.com/chrismccord/phoenix_live_view_example/pull/27

OR, I figure out how to get my local copy running without that change.